docs/03-concepts/05-topology.md

```diff
@@ -18,7 +18,7 @@ Note that both types of :worker:workers: as well as external clients are roles a
 ![Cadence Architecture](https://user-images.githubusercontent.com/14902200/160308507-2854a98a-0582-4748-87e4-e0695d3b6e86.jpg)
 
 
-At the core of Cadence is a highly scalable multitenant service. The service exposes all of its functionality through a strongly typed [gRPC API](https://github.com/cadence-workflow/cadence-idl/tree/master/proto/cadence-workflow/cadence/api/v1). A Cadence cluster include multiple services, each of which may run on multiple nodes for scalability and reliablity:
+At the core of Cadence is a highly scalable multitenant service. The service exposes all of its functionality through a strongly typed [gRPC API](https://github.com/cadence-workflow/cadence-idl/tree/master/go/proto/api/v1). A Cadence cluster include multiple services, each of which may run on multiple nodes for scalability and reliablity:
```

Comment:
```suggestion
At the core of Cadence is a highly scalable multitenant service. The service exposes all of its functionality through a strongly typed [gRPC API](https://github.com/cadence-workflow/cadence-idl/tree/master/proto/uber/cadence/api/v1). A Cadence cluster include multiple services, each of which may run on multiple nodes for scalability and reliablity:
```

This would be the correct link, since go/proto links to the code generated from the proto definitions, and not the definitions themselves.

It's interesting that we still have `uber/` here as the directory after the GH org migration from `uber` to `cadence-workflow`, maybe let's check with the team?